Sources/MongoSwift/ReadWriteConcern.swift

```diff
@@ -114,3 +113,175 @@ extension ReadConcern: Equatable {
         return lhs.level == rhs.level
     }
 }
+
+/// A class to represent a MongoDB write concern.
+public class WriteConcern: Codable {
+
+    /// A pointer to a mongoc_write_concern_t
+    internal var _writeConcern: OpaquePointer?
+
+    /// An option to request acknowledgement that the write operation has propagated to specified mongod instances.
+    public enum W: Codable, Equatable {
```

Comment:
Ah, another question that came up. Why is this declared within WriteConcern when ReadConcernLevel is declared outside of ReadConcern?

Reply:
https://jira.mongodb.org/browse/SWIFT-123